package.json

```diff
@@ -23,13 +22,12 @@
     ]
   },
   "scripts": {
-    "prebuild": "npm run clean",
+    "prebuild": "node -e \"var s=require('shelljs'),d=['generators','es5-check','es5-test'];s.rm('-rf',d);s.mkdir('-p',d)\"",
```

Comment:
I made it a little cleaner by making a clean.js file:
```js
var s = require('shelljs'),
  d = process.argv.slice(2);
s.rm('-rf', d);
s.mkdir('-p', d);
```
` "prebuild": "shjs clean.js generators es5-check es5-test",`